src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -124,18 +138,16 @@ TetMesh::TetMeshOperationExecutor::TetMeshOperationExecutor(
 
     // update hash on all tets in the two-ring neighborhood
     simplex::SimplexCollection hash_update_region(m);
-    for (const simplex::Simplex& v : edge_closed_star.simplex_vector(PrimitiveType::Vertex)) {
-        const simplex::SimplexCollection v_closed_star =
-            simplex::top_dimension_cofaces(m_mesh, v, false);
-        hash_update_region.add(v_closed_star);
+    for (const simplex::Simplex& v : edge_closed_star_vertices.simplex_vector()) {
+        simplex::top_dimension_cofaces(v, hash_update_region, false);
```

Comment:
I'd rather pass a mesh and a vector for top dimension cofaces than use the SimplexCollection interface - it's just making code more opaque here (the mesh is implicit now, is there ever a use of this interface where you don't pass in false?)

Reply:
The vector is cleaned right after that loop. I think for now it's simpler to use the SimplexCollection but this code needs to be cleaned up in general.